token-validation-core/src/main/java/no/nav/security/token/support/core/api/Protection.java

```diff
@@ -0,0 +1,9 @@
+package no.nav.security.token.support.core.api;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Protection {
```

Comment:
Not sure about Protection as the name here. I would prefer something along the lines of `RequiredIssuers` and then required claims are implicit by the reference to `ProtectedWithClaims`.